proto/internalapi/central/cluster_status.proto

```diff
@@ -21,4 +21,5 @@ message ClusterStatusUpdate {
 message RawClusterHealthInfo {
     storage.CollectorHealthInfo collector_health_info                   = 1;
     storage.AdmissionControlHealthInfo admission_control_health_info    = 2;
+    storage.LocalScannerHealthInfo local_scanner_health_info            = 3;
```

Comment:
Since I see it was adjusted in the proto below, should we align these variables, too?